code/datums/outfits/outfit.dm

```diff
@@ -40,6 +41,10 @@
 	var/tablet = /obj/item/modular_computer/handheld/preset/civilian
 	var/wristbound = /obj/item/modular_computer/handheld/wristbound/preset/pda/civilian
 
+	var/allow_headset_choice = FALSE
```

Comment:
What's this for?

Reply:
Just in case you need to disable it for any reason (like there not being a bowman headset or you prefer people not to spawn with NO headset)

I'm just following backbag and pda choice, I thought it'd be useful